### ISSUE TITLE
Harmonise AirTable registry of the Ersilia Model Hub and available repositories

### ISSUE THREAD

**Issue by miquelduranfrigola** [ersilia-os/ersilia#917]
Currently if, for some reason, a model GitHub repository is deleted, the AirTable registry is not aware of it. Likewise, if a record is deleted from AirTable, the repository remains in GitHub.

It would be nice to develop a GitHub Action that checks coherence between available GitHub repositories and records in AirTable. At the end of the action, if any incosistency is found, we could send a message to Slack.

**Comment by miquelduranfrigola** [comment]
Thought: this may be moved to the [ersilia-stats](https://github.com/ersilia-os/ersilia-stats) repository, if needed.

**Comment by GemmaTuron** [comment]
This is linked to #768

**Comment by GemmaTuron** [comment]
I don't think this should go to the ersilia-stats, as this is a maintenance issue not a stats issue

**Comment by miquelduranfrigola** [comment]
I agree. I am just worried that we will have too many actions in `ersilia` and this may become difficult to manage/browse. We could create an `ersilia-maintenance` repo for these kinds of tasks, although I am not convinced either.

**Comment by miquelduranfrigola** [comment]
**Update** for the upcoming Harvard T4SG round, we have suggested the creation of an `ersilia-maintenance` repository.

**Comment by GemmaTuron** [comment]
Ok, we have an ersilia-maintenance repository now, so is this problem being tackled there already? and if not, can we transfer the issue there so that it gets in the right repo?

**Comment by DhanshreeA** [comment]
This problem is not being tackled, as evidenced by us having to delete/add entries manually in Airtable. Transferring this issue to ersilia-maintenance, and assigning myself.
